StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -77,7 +80,11 @@ extension PaymentSheetVerticalViewController: BottomSheetContentViewController {
 
     func didTapOrSwipeToDismiss() {
         // TODO
-        delegate?.flowControllerViewControllerShouldClose(self, didCancel: true)
+        if isFlowController {
+            flowControllerDelegate?.flowControllerViewControllerShouldClose(self, didCancel: true)
+        } else {
+            paymentSheetDelegate?.paymentSheetViewControllerDidCancel(self)
+        }
```

Comment:
Is there any harm in calling both of them? One should be nil so it'd no-op right? Then we could remove `isFlowController` all together.

Reply:
No harm in calling both. We'll need `isFlowController` or something similar later though.